### ISSUE TITLE
Stuck: At project creation, cant exit or login

### ISSUE THREAD

**Issue by gregg-cbs** [codesandbox/codesandbox-client#7014]
# 🐛 bug report

## Preflight Checklist

- [x] I have read the [Contributing Guidelines][contributing] for this project.
- [x] I agree to follow the [Code of Conduct][code_of_conduct] that this project
      adheres to.
- [x] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem
On project creation and choosing a start project/template I am getting an error to login but it is not possible for me to do that because I cant exit the create project screen.

## How has this issue affected you? What are you trying to accomplish?
Trying to create a server project but cannot because I need to be logged in.

### To Reproduce
Go to https://codesandbox.io , scroll a little and click 'get started for free', in the template search, search for 'Nuxt 3', click any nuxt template to start and notice the errors at the bottom of your screen. Now assume you want to login or get away from this template choosing screen... how do you?

### Video recording of incident
https://user-images.githubusercontent.com/57114335/195291000-56a8d1df-de4a-4093-b950-4c885c2f93be.mp4

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |      browser        |
| Browser          |        firefox      |
| Operating System |        windows      |

[code_of_conduct]: https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md
[contributing]: https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md

**Comment by tristandubbeld** [comment]
Hey @gregg-cbs, once again thanks for reporting an issue :)

We're discussing on how to improve the UX by e.g. showing the sign in button in this tab and I'll try to get started on this ASAP. In the meantime you should be able to login through the "Import form GitHub" tab. There's a login button there. Thanks again for your help, we really appreciate it!

**Comment by gregg-cbs** [comment]
No problem, i am happy to help and report. Glad its received well. Look forward to seeing the update.

**Comment by tristandubbeld** [comment]
The changes should be live @gregg-cbs. We're showing a sign in button in the notification as well as in more tabs. Thanks again for your feedback!

**Comment by gregg-cbs** [comment]
I see its live, thanks for making that update but theres a spot you missed.

Because you are sometimes showing the sign in button in the notification and other times in the templates modal tab content (like on the Cloud templates tab), then sometimes you dont see the sign in button at all.

"Quick start" tab -> "NextJS" template... no login button:
![image](https://user-images.githubusercontent.com/57114335/195804375-31cf5835-049a-497f-8092-fba20450aabe.png)

I think you should put the login button next to the search input so you dont have multiple login buttons and cases where it might not exist.
![image](https://user-images.githubusercontent.com/57114335/195806123-bb0207a0-4c3c-4dc5-a6cb-85e82059d5d4.png)

I was originally expecting to see it in the side nav like this:
![image](https://user-images.githubusercontent.com/57114335/195806710-9ba0dbad-6178-4fae-a8cf-c1b031c5f3d8.png)

**Comment by tristandubbeld** [comment]
You're right! I was unaware of a change that happened  when checking if we should show a sign in button on the error message. I've created a PR to make sure we show a sign in button in the error message again.

> "Quick start" tab -> "NextJS" template... no login button:

Since it's not required to sign in for non-beta templates we don't show a sign in button on every tab. The cloud templates tab requires the user to sign in to create a cloud sandbox (for now). As a "quick convenience" I've added it there because there are no "anonymous user templates" in that tab.

> I think you should put the login button next to the search input [...]

> or I was originally expecting to [...]

Yep good suggestions, we're looking into the best way to tackle this. I'll discuss if this is something we want and if it doesn't disrupt future plans.
